manifests/machineconfigcontroller/controllerconfig.yaml

```diff
@@ -4,10 +4,4 @@ metadata:
   name: machine-config-controller
   namespace: {{.TargetNamespace}}
 spec:
-  clusterDNSIP: {{.ControllerConfig.ClusterDNSIP}}
-  cloudProviderConfig: {{.ControllerConfig.CloudProviderConfig}}
-  clusterName: {{.ControllerConfig.ClusterName}}
-  platform: {{.ControllerConfig.Platform}}
-  baseDomain: {{.ControllerConfig.BaseDomain}}
-  etcdCAData: {{.ControllerConfig.EtcdCAData | toString | b64enc}}
-  rootCAData: {{.ControllerConfig.RootCAData | toString | b64enc}}
+{{toYAML .ControllerConfig | toString | indent 2}}
```

Comment:
How is base64 encoding for the CAs handled now?

Reply:
they were `[]byte` and yaml marshalling automatically converts `[]byte` to `base64encoded string`